fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/A380_EXTERIOR.xml

```diff
@@ -35,27 +35,21 @@
             </UseTemplate>
 
             <!-- LH Body Gear -->
-            <UseTemplate Name="ASOBO_GT_Anim">
+            <UseTemplate Name="ASOBO_GEAR_Back_Template">
```

Comment:
This doesn't look right. Aren't we supposed to use our own LVars?

cc @crocket63

Reply:
Oh damnit, I'm very sorry, I thought we were working off the latest master here. Will revert these changes, and only leave the door clickspot in.

Reply:
@crocket63 Is everything in order now?